Amazon.IonObjectMapper/IonSerializer.cs

```diff
@@ -186,6 +189,12 @@ public interface IIonSerializerFactory
     /// </summary>
     public abstract class IonSerializerFactory<T> : IIonSerializerFactory
     {
+        /// <summary>
+        /// Create custom IonSerializer with customContext option.
+        /// </summary>
+        /// <returns>

```

Comment:
Should also have the `<param>` tags for the input parameters.

Reply:
Added.